### ISSUE TITLE
Support multiple profiles

### ISSUE THREAD

**Issue by jornane** [geteduroam/ionic-app#27]
Right now the app supports only one profile to be installed at a time. There are cases where you would have multiple profiles installed, e.g. when working at multiple institutions with eduroam, when testing IdP or when using the app for a different network than eduroam.

Additionally, the eap-config format supports multiple EAP `<EAPIdentityProvider>` entries in the `<EAPIdentityProviderList>`. I think the app currently only uses the first one. There are as far as I know currently no cases where an eap-config does contain more than one provider.

We need a GUI overhaul to support this, so this is not a priority.

**Comment by spaetow** [comment]
Yes, this would be good to have (we currently cannot test the app with our alternative config because of this). 👍
